src/cargo/ops/common_for_install_and_uninstall.rs

```diff
@@ -545,7 +546,15 @@ impl InstallInfo {
 
 /// Determines the root directory where installation is done.
 pub fn resolve_root(flag: Option<&str>, gctx: &GlobalContext) -> CargoResult<Filesystem> {
-    let config_root = gctx.get_path("install.root")?;
+    let config_root = gctx
+        .get::<OptValue<ConfigRelativePath>>("install.root")
```

Comment:
Just FYI, this can be `Option<ConfigRelativePath>>`. We don't use the definition here, so this can be simplified.